src/features/histories-slice.js

```diff
@@ -1,7 +1,7 @@
 import { createSlice } from '@reduxjs/toolkit';
 
 const initialState = {
-  history: [
+  histories: [
```

Comment:
must not contain mock data, please do not forget to remove it once start implementing this slice